### PR TITLE
fix: removing health endpoint, params changes

### PR DESCRIPTION
**Pull request by psturc**
### Motivation
https://issues.jboss.org/browse/AGMS-182

Screenshot 2019-03-12 at 10 21 19
Screenshot 2019-03-12 at 10 21 30

**Update:** I have also fixed passing the "graphql_endpoint" env var to the server URL ("/graphql" was hardcoded before). To verify:

1. Login to https://master.psturc-620a.openshiftworkshop.com/console/project/agms-182/overview
2. Provision Data Sync (and change "graphq endpoint" variable to something else)
3. Go to MDC and bind the datasync
4. See the mobile-services.json config and verify that `url` and `websocketUrl` contain changed graphql endpoint

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -53,11 +48,13 @@ plans:
...
         required: True
         title: Sync App Port
+        default: 4000
       - name: SYNC_APP_GRAPHQL_ENDPOINT
-        description: "(default value: 'graphql') GraphQL Endpoint of your Sync Application"
+        description: "GraphQL Endpoint of your Sync Application"
         type: string
         required: False
         title: Sync App GraphQL Endpoint
+        default: 'graphql'
```
Comment (darahayes) [inline apb.yml]: Question: Is there a reason we expect users input a string like `metrics` and then we're adding the `/` to the front to make `/metrics`? 

I'm just wondering is this a standard thing in other APBs or in the wider OpenShift world? To me it's pretty unintuitive. On a couple of occasions, I have put in a url like `/health` which resulted in `//health` and that caused failing deployments.

**Comment by psturc** [comment]
@wei-lee I think adding a "/" before the endpoint name makes more sense as Dara mentioned, so I will unify it for all "endpoint" parameters.
Making DB parameters optional is definitely a timesaver for a user

**Comment by psturc** [comment]
Ready for re-review

**Comment by wei-lee** [comment]
LGTM

**Pull request merged**